Px.Utils.UnitTests/Validation/DataValidationTests/DataNumberValueValidator.cs

```diff
@@ -0,0 +1,53 @@
+﻿using PxUtils.Validation;
+using PxUtils.Validation.DataValidation;
+
+namespace Px.Utils.UnitTests.Validation.DataValidationTests;
```

Comment:
@jsaarimaa Do we still want to keep the namespaces consistent using "PxUtils" instead of "Px.Utils" or did we agree that it's too much effort to bother for no sufficient reason?

Reply:
Other classes have their namespaces defined in a way where the namespace statement encloses the class. Would be preferable to have all classes defined in the same way

Reply:
will change to block scoped namespace